lib/embed/embed.dart

```diff
@@ -533,11 +536,34 @@ class PlaygroundMobile {
       _editProgress.hidden(false);
     }
 
+    if (_requestStored()) {
+      new Future(() {
+        CompileResponse response = _storedResponse;
+        if (executionService != null) {
+          return executionService.execute(
+              _context.htmlSource, _context.cssSource, response.result);
+        }
+      }).catchError((e) {
+        _showOutput('Error compiling to JavaScript:\n${e}', error: true);
+        _showError('Error compiling to JavaScript', '${e}');
```

Comment:
Is it expected that _runbutton will be disabled if this path gets executed? That seems unfortunate as it means that in the presence of a server error the user is not going to get the chance to request another compile